ports/qt5-base/cmake/configure_qt.cmake

```
@@ -142,7 +142,7 @@ function(configure_qt)
         string(REPLACE "${CMAKE_CURRENT_INSTALLED_DIR_PATH}" "\${CURRENT_INSTALLED_DIR}" _contents ${_contents})
         #string(REPLACE "HostPrefix=\${CURRENT_PACKAGES_DIR}" "HostPrefix=\${CURRENT_INSTALLED_DIR}" _contents ${_contents})
         string(REPLACE "[EffectivePaths]\nPrefix=..\n" "" _contents ${_contents})
-        string(REPLACE "[EffectiveSourcePaths]\nPrefix=${_csc_SOURCE_PATH}\n" "" _contents ${_contents})
+        string(REGEX REPLACE "[\[]EffectiveSourcePaths[\]][\r]?\nPrefix=[^\r\n]+[\r]?\n" "" _contents ${_contents})
```

```suggestion
        string(REGEX REPLACE "\\[EffectiveSourcePaths\\]\\r?\\nPrefix=[^\\r\\n]+\\r?\\n" "" _contents ${_contents})
```

I think the escaping here is confused because `\[` for example is what you wanted to pass to the regex engine, but `\` is special in cmake literals, so you effectively passed, for example, `[]]` into the regex engine here.

`\\` causes `\\` to become `\` when seen by the regex engine.

---

Yes I confirmed that works.

---

`\r` and `\n` should not be translated, because they are indeed matched here.